bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4254,6 +4254,40 @@
             INSERT INTO role_role(child_role, parent_role) VALUES ('Appointments:FullAccess', 'Appointments:ManageAppointments');
         </sql>
     </changeSet>
+    <changeSet id="201802211437" author="ningosi">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+                SELECT COUNT(*) FROM global_property where property = 'patientIdentifierSearch.matchMode';
+            </sqlCheck>
+        </preConditions>
+        <comment>Specify how patient identifiers are matched while searching for a patient</comment>
```

Comment:
This comment is not helpful since it provides no additional info that can't be inferred from "patientIdentifierSearch.matchMode". Instead say here _why_ we're making the change, e.g. "change from default OpenMRS patient identifier matching behavior to search anywhere"